reference/versions/0.5.0/reference.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
I understand that a new analysis requires two PR, first one to implement the node and test it isolated and second one to integrate it into analysis workflow. right?